### DISCUSSION TITLE
I can't see my project information in Wallet Connect Cloud

### DISCUSSION THREAD

**Discussion by caioflores** [WalletConnect/walletconnect-monorepo#4316] [Developer Support]
I created a wallet project a while ago and it's already in Production for a while. But now whenever I try to log in the cloud platform it keeps infinite loading and I can't see my project information (image)
![Screenshot 2024-03-11 at 16 30 54](https://github.com/WalletConnect/walletconnect-monorepo/assets/4183877/31192d17-a16f-4ccd-b5df-1953bd3edf43)

**Comment by glitch-txs** [comment]
Hi, do you see any errors on the console?

**Reply by caioflores** [reply]
Just this

![Screenshot 2024-03-12 at 18 55 00](https://github.com/WalletConnect/walletconnect-monorepo/assets/4183877/9c42caee-062e-453b-aadd-02116f9abfd1)

**Comment by Cali93** [comment]
Hey @caioflores, we didn't find any project that belongs to that address.
Could you provide your `projectId` used in prod ? 
Is it possible that you used email + password as the authentication method or another address ?

**Reply by caioflores** [reply]
It might be the case, but I also tried to recover my password and it's not working.

The project id is 08259637f7088cf772cc449b0a47d07e

**Reply by Cali93** [reply]
I can confirm that you used your work email to create the account.
You can try [resetting your password here](https://cloud.walletconnect.com/forgot-password).

**Reply by caioflores** [reply]
I tried that, but I'm getting this error 

![Screenshot 2024-03-13 at 14 27 01](https://github.com/WalletConnect/walletconnect-monorepo/assets/4183877/2c155d07-3b77-4a13-965c-5f2e1e256cad)

**Reply by caioflores** [reply]
Any news?

**Reply by Cali93** [reply]
Hey @caioflores, 

Thanks for your patience. The issue should be fixed.

Please go through the entire [forgot password flow](https://cloud.walletconnect.com/forgot-password) again, and make sure that the link you receive in the reset password email isn't already used.
Some companies have anti-spam filters that click on links, making the reset password link invalid. If that's the case you should ask your IT department to whitelist our email address.

Please let us know if it worked or if you're still having issues.

**Reply by caioflores** [reply]
I'm still facing the same error, whenever I tap the email link I'm redirected to this: https://cloud.walletconnect.com/reset-password#error=unauthorized_client&error_code=401&error_description=Email+link+is+invalid+or+has+expired

So it seems that I'm receiving a link that is not working. Could it be related to any kind of "anti-spam" filter on our side?

**Reply by Cali93** [reply]
Yes it's very likely due to the anti-spam filter that makes the link invalid.

**Reply by caioflores** [reply]
Yes, the issue was on our side, now I'm able to access it, thanks!
